AppEngine/src/main/resources/views/index.hbs

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
diff HTMLs from live and new

Reply:
[net.twisterrob.travel.zip](https://github.com/TWiStErRob/net.twisterrob.travel/files/13481052/net.twisterrob.travel.zip)

looks reasonable, not sure why date format changed, not visible in Chrome.